safe/gui/tools/print_report_dialog.py

```diff
@@ -65,36 +72,96 @@ def __init__(self, impact_function, iface, dock=None, parent=None):
+            impact_report_pdf_component['key']:
+                self.impact_summary_checkbox,
+            action_checklist_pdf_component['key']:
+                self.action_checklist_checkbox,
+            analysis_provenance_details_pdf_component['key']:
+                self.provenance_checkbox,
+            infographic_report['key']:
+                self.infographic_checkbox
+        }
+
+        # setup checkboxes, all checkboxes are checked by default
+        for checkbox in self.all_checkboxes.values():
+            checkbox.setChecked(True)
+
+        # override template is selected by default
+        self.default_template_radio.setChecked(True)
+
+        self.is_multi_exposure = isinstance(
+            self.impact_function, MultiExposureImpactFunction)
+
+        override_template_found = None
+        if self.is_multi_exposure:
+            self.override_template_radio.setEnabled(False)
+            self.override_template_label.setEnabled(False)
+            self.override_template_found_label.setEnabled(False)
+            # below features are currently not applicable for multi-exposure IF
+            self.action_checklist_checkbox.setEnabled(False)
+            self.action_checklist_checkbox.setChecked(False)
+            self.provenance_checkbox.setEnabled(False)
+            self.provenance_checkbox.setChecked(False)
+            self.infographic_checkbox.setEnabled(False)
+            self.infographic_checkbox.setChecked(False)
+        else:
+            # search for available override template
+            hazard_type = definition(
+                self.impact_function.provenance['hazard_keywords'][
+                    'hazard'])
+            exposure_type = definition(
+                self.impact_function.provenance['exposure_keywords'][
+                    'exposure'])
+            # noinspection PyArgumentList
+            custom_template_dir = join(
+                QgsApplication.qgisSettingsDirPath(), 'inasafe')
+            if exists(custom_template_dir) and hazard_type and exposure_type:
+                for filename in listdir(custom_template_dir):
+
+                    file_name, file_format = splitext(filename)
+                    if file_format[1:] != (
+                            QgisComposerComponentsMetadata.OutputFormat.QPT):
+                        continue
+                    if hazard_type['key'] in file_name and (
+                                exposure_type['key'] in file_name):
+                        override_template_found = filename
+
+        if override_template_found:
+            string_format = tr('*Template override found: {template_path}')
+            self.override_template_found_label.setText(
+                string_format.format(template_path=override_template_found))
+
         # additional buttons
-        self.button_print_pdf = QtGui.QPushButton(self.tr('Print as PDF'))
+        self.button_print_pdf = QtGui.QPushButton(self.tr('Open as PDF'))
```

Comment:
Avoid to use `self.tr`, just use `tr` to reduce the number of translation. Use context if you want to make it translated differently.